napari/utils/io.py

```diff
@@ -1,16 +1,85 @@
 import os
+import re
 
 from glob import glob
 from pathlib import Path
 
 import numpy as np
-from skimage import io
-from skimage.io.collection import alphanumeric_key
 
 from dask import delayed
 from dask import array as da
 
 
+def imsave(filename: str, data: np.ndarray):
+    """custom imaplementation of imread to avoid skimage dependecy"""
+    ext = os.path.splitext(filename)[1]
+    if ext in [".tif", "tiff"]:
+        import tifffile
+
+        tifffile.imsave(filename, data)
+    else:
+        import imageio
+
+        imageio.imsave(filename, data)
+
+
+def imread(filename: str):
+    """custom imaplementation of imread to avoid skimage dependecy"""
```

Comment:
```suggestion
    """Custom implementation of imread to avoid skimage dependency.

    Parameters
    ----------
    filename : string
        The path from which to read the image.

    Returns
    -------
    data : np.ndarray
        The image data.
    """
```